lib/Echidna/ABI.hs

```diff
@@ -62,21 +62,19 @@ encodeSig n ts = n <> "(" <> T.intercalate "," (map abiTypeSolidity ts) <> ")"
 genSize :: MonadGen m => m Int
 genSize = (8 *) <$> Gen.enum 1 32
 
-genWord256 :: MonadGen m => m Word256
-genWord256 = let w64 = Gen.word64 $ constant minBound maxBound in
-  liftM2 Word256 (liftM2 Word128 w64 w64) (liftM2 Word128 w64 w64)
-
 genAbiAddress :: MonadGen m => m AbiValue
 genAbiAddress = let w64 = Gen.word64 $ constant minBound maxBound in
   fmap AbiAddress . liftM2 Word160 Gen.enumBounded $ liftM2 Word128 w64 w64
 
 genAbiUInt :: MonadGen m => Int -> m AbiValue
-genAbiUInt n = AbiUInt n . scale <$> genWord256 where
-  scale = if n == 256 then id else flip mod (2^n)
+genAbiUInt n = AbiUInt n . fromInteger <$> genUInt (toInteger n)
```

Comment:
instead of having a unary `genUInt`, `n` is in scope in the local def so why not just have it constant?